include/padscore/kpad.h

```diff
@@ -271,6 +278,21 @@ KPADReadEx(KPADChan chan,
            uint32_t size,
            KPADError *error);
 
+/**
+ * Sets MotionPlus for the controller in specified mode
+ * \sa
+ * - KPADMotionPlusMode
+ */
+void
+KPADEnableMpls(KPADChan channel,
+               uint8_t mode);
```

Comment:
This could be:
```suggestion
/**
 * Sets MotionPlus for the controller in specified mode
 * 
 * \param mode
 * The MotionPlus mode which should be used
 */
void
KPADEnableMpls(KPADChan channel,
               KPADMotionPlusMode mode);
```

Reply:
Doesn't matter in this case since both the 4 and 1 byte value will be placed in a 32-bit register.